app/views/admin/claims/_info_panel_identity_unconfirmed.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="banner-temporary-message-without-action">
+  <h2 class="govuk-body-l">
+    The claimant did not complete GOV.UK Verify
+  </h2>
+
+  <p class="govuk-body">
+    Refer to the <%= link_to "Confirming a claimant's identity playbook", confirming_identity_playbook_url, class: "govuk-link" %>
+    and contact <%= school.name %> on <strong><%= school.phone_number %></strong>.
+  </p>
+
+  <p class="govuk-body">
+    Once you have completed all the steps outlined in the playbook, you can
+    <%= link_to "approve or reject this claim", "#new_check", class: "govuk-link" %>.
```

Comment:
I know this `new_check` ID is autogenerated by Rails, but I'm wondering if it might be clearer if we link to an ID we define ourselves, to guard against the risk of something inside Rails changing and the form IDs being generated differently

Reply:
Yep, good shout, better to be explicit, especially as [`form_with`](https://apidock.com/rails/ActionView/Helpers/FormHelper/form_with) doesn't magically set an `id` attribute.